src/java.base/share/classes/java/lang/ThreadGroup.java

```diff
@@ -555,17 +555,6 @@ private int enumerate(ThreadGroup[] list, int i, boolean recurse) {
         return i;
     }
 
-    /**
-     * Throws {@code UnsupportedOperationException}.
-     *
-     * @deprecated This method was originally specified to stop all threads in
-     *             the thread group. It was inherently unsafe.
-     */
-    @Deprecated(since="1.2", forRemoval=true)
-    public final void stop() {
-        throw new UnsupportedOperationException();
-    }
```

Comment:
This must stay; see https://github.com/ibmruntimes/openj9-openjdk-jdk23/commit/d987abcc506196944c49f1293ba4cd9eb0d3ee81.

Reply:
Reverted OpenJDK change.